frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx

```diff
@@ -97,7 +98,7 @@ export function InsightDisplayConfig({
                     <span className="filter">
                         <span className="head-title-item">Date range</span>
                         <InsightDateFilter
-                            defaultValue={getFormattedLastWeekDate()}
+                            defaultValue={currentFormattedDateRange}
```

Comment:
Why are we using defaultValue here in the first place instead of binding value?

Reply:
Hey Karl what do you mean by binding value here?

Reply:
I'm not Karl, but if we have `defaultValue` anywhere, it's a non-binding value --> only read the first time the component (insight date filter) renders. If the value of `defaultValue` ever changes, the component won't pick this up.

If we have `value` and `onChange`, we have a bound value --> any time the value changes, the component updates.

It's preferable to have bound values when possible, as this limits random surprises.

Reply:
got it thanks @mariusandra. The `defaultValue` was already there before my changes which is why I carried it over. If you look inside of `DateFilter.tsx` you'll find that the `currKey` (current value) reacts on `dateFrom`, `dateTo`, and `defaultValue` to determine which date to show in the select input. The `dateFrom` and `dateTo` are the bound values, and `defaultValue` is a string that you can fall back on if `dateFrom` and `dateTo` and invalid. In this case `defaultValue` updating seems to update the component text as well. 

With that being said though, the name `defaultValue` is misleading and might be better named to `fallbackValue` since this homebaked component updates on `defaultValue` changes. I think it'll also be cleaner to pass in a set of default props into `insightDateFilterLogic` instead of cramming it all into a selector. Don't want to block you so I'll clean this up now